packages/suite/src/views/wallet/coinmarket/sell/offers/Offers/List/Quote/index.tsx

```diff
@@ -0,0 +1,272 @@
+    width: 100%;
+    min-height: 150px;
+    padding-bottom: 16px;
+    background: ${props => props.theme.BG_WHITE};
+`;
+
+const TagRow = styled.div`
+    display: flex;
+    min-height: 30px;
+`;
+
+const Tag = styled.div`
+    margin-top: 10px;
+    height: 35px;
+    margin-left: -20px;
+    border: 1px solid tan;
+    text-transform: uppercase;
+`;
+
+const Main = styled.div`
+    display: flex;
+    margin: 0 30px;
+    justify-content: space-between;
+    padding-bottom: 20px;
+    border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        flex-direction: column;
+    }
+`;
+
+const Left = styled.div`
+    display: flex;
+    font-size: ${variables.FONT_SIZE.H2};
+`;
+
+const Right = styled.div`
+    display: flex;
+    justify-content: flex-end;
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        justify-content: center;
+        padding-top: 10px;
+    }
+`;
+
+const Details = styled.div`
+    display: flex;
+    min-height: 20px;
+    flex-wrap: wrap;
+    padding: 10px 30px;
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        flex-direction: column;
+    }
+`;
+
+const Column = styled.div`
+    display: flex;
+    padding: 10px 0;
+    flex: 1;
+    flex-direction: column;
+    justify-content: flex-start;
+`;
+
+const Heading = styled.div`
+    display: flex;
+    text-transform: uppercase;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    padding-bottom: 9px;
+`;
+
+const StyledButton = styled(Button)`
+    width: 180px;
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        width: 100%;
+    }
+`;
+
+const Value = styled.div`
+    display: flex;
+    align-items: center;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const Footer = styled.div`
+    margin: 0 30px;
+    padding: 10px 0;
+    padding-top: 23px;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    border-top: 1px solid ${props => props.theme.STROKE_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const ErrorFooter = styled.div`
+    display: flex;
+    margin: 0 30px;
+    padding: 10px 0;
+    border-top: 1px solid ${props => props.theme.STROKE_GREY};
+    color: ${props => props.theme.TYPE_RED};
+`;
+
+const StyledIcon = styled(Icon)`
+    padding-top: 8px;
+`;
+
+const IconWrapper = styled.div`
+    padding-right: 3px;
+`;
+
+const ErrorText = styled.div``;
+
+const StyledQuestionTooltip = styled(QuestionTooltip)`
+    padding-left: 4px;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+export function getQuoteError(quote: SellFiatTrade, amountInCrypto: boolean) {
+    if (quote.error) {
+        if (amountInCrypto) {
+            if (quote.minCrypto && Number(quote.cryptoStringAmount) < quote.minCrypto) {
```

Comment:
`Number(quote.cryptoStringAmount)` conversion is used multiple times in this code block, could be assigned as const?

Reply:
done